src/go/rpk/pkg/cli/cmd/cluster/partitions/status.go

```diff
@@ -0,0 +1,68 @@
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewBalancerStatusCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "balancer-status",
+		Short: "Queries cluster for partition balancer status",
+		Long: `Queries cluster for partition balancer status:
+
+FIELDS DESCRIPTION:
+
+    Status:                        Either off, ready, starting, in_progress or stalled.
+    Seconds Since Last Tick:       The last time the partition balancer ran.
+    Current Reassignments Count:   Current number of partition reassignments in progress.
+    Unavailable Nodes:             The nodes that have been unavailable after a time set by 'partition_autobalancing_node_availability_timeout_sec' property.
+    Over Disk Limit Nodes:         The nodes that surpassed the threshold of used disk percentage set by 'partition_autobalancing_max_disk_usage_percent' property.
```

Comment:
Should I say: set by <...> **cluster** property?

Reply:
I like both,
also needs the:
`set by the ' ' cluster property`